packages/superset-ui-chart/src/query/Metric.ts

```diff
@@ -1,5 +1,5 @@
 import { Column } from './Column';
-import FormData from './FormData';
+import { FormData } from './FormData';
 
 export const LABEL_MAX_LENGTH = 43;
 
```

Comment:
Why `.filter(metric => metric)` at line 60? (sorry github won't let me give a comment on that line).

Also, I found a bug in my original implication when putting the demo yesterday (metrics in form_data can appear as arrays of metrics). I can put up a fix against this branch.

Reply:
Filter the undefined field